### PR TITLE
Most UI done (page /characters)

### PR DESCRIPTION
**Pull request by Keitio**

### CODE REVIEW

File: src/pages/characters.tsx [hunk 1]
```diff
@@ -1,79 +1,114 @@
...
+          onClick={() => setCostume(costume)}>
+          {costume.name.en}
         </div>
-      </div> */}
+      ))}
+    </div>
+  )
+}
+
+export default function Home() {
```
Comment (KeziahMoselle) [inline src/pages/characters.tsx:95]: I forgot to rename this component, we can probably rename it to CharactersPage or something along those lines?

File: src/pages/characters.tsx [hunk 2]
```diff
@@ -1,79 +1,114 @@
...
+
+
+  const [currentCostume, setCostume] = useState(
+    typedCharacters.values().next().value[0] as CostumeInfo
+  )
+
+  return (
+    <Layout>
+      <Head>
+        <title>NieR Re[in] Global Guide & Database</title>
```
Comment (KeziahMoselle) [inline src/pages/characters.tsx:105]: Can we add "Characters -" at the start ?

**Review by KeziahMoselle — APPROVED** [review]

File: src/pages/characters.tsx (hunk 1 above)
Comment (Keitio) [reply src/pages/characters.tsx:95]: I'll do that, but it's not important because it's the default export

Comment (KeziahMoselle) [reply src/pages/characters.tsx]: It will show "Home" as the component name within the React Devtools so still a nice thing to fix imo

**Pull request merged**